enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/rest/RestOnboardingProviderConfiguration.java

```diff
@@ -47,4 +50,17 @@ class RestOnboardingProviderConfiguration {
     private String httpBasicAuthUsername;
 
     private String httpBasicAuthPassword;
+
+    private Header correlationHeader = new Header("X-Correlation-Id");
```

Comment:
I believe we should make the correlation header name configurable. There should be no hardcoded constants of header names in the code. See e.g.
https://github.com/wultra/powerauth-server/blob/85b5e16ec663f44717d0dd76b5566cb1d301c4dc/powerauth-java-server/src/main/resources/application.properties#L118

Reply:
It is configurable. I changed the default names to constant to make it clearer.